client/my-sites/stats/annual-highlights-section/index.tsx

```diff
@@ -58,6 +71,21 @@ export default function AnnualHighlightsSection( { siteId }: { siteId: number }
 	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
 	const viewMoreHref = siteSlug ? `/stats/annualstats/${ siteSlug }` : null;
 
+	const previousYear = insights?.years?.find( ( y ) => y.year === ( year - 1 ).toString() );
```

Comment:
`insights.years` sometimes skips years without data. For example, this is what it looks like for site 27540269 (@keoshi's site! 🙂):

```js
[
  ...
  { "year": "2016", ... },
  { "year": "2017", ... },
  { "year": "2018", ... },
  { "year": "2020", ... },
  { "year": "2022", ... }
]
```

Notice how it skips `2019` and `2021`.